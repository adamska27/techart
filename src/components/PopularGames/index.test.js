import React from 'react';
import { shallow } from 'enzyme';

import Component from './index';

describe('PopularGames', () => {
  it('should return the loader', () => {
    expect(
      shallow(<Component isFetching={true} />).contains(<p>Loading</p>)
    ).toBe(true);
  });
});
