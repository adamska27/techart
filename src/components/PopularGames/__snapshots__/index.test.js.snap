// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopularGames should match the snapshot 1`] = `
<ul
  className="sc-bwzfXH ksqnaS"
/>
`;
